src/config/replicator.rst

```diff
@@ -103,10 +103,17 @@ Replicator Database Configuration
 
     .. config:option:: retries_per_request
 
-        If a request fails, the replicator will retry it up to N times::
+        .. versionchanged:: 2.1.1
+
+        If a request fails, the replicator will retry it up to N times. The
+        default value for N is 5 (before version 2.1.1 it was 10). The requests
+        are retried with an doubling exponenital backoff starting at 0.25
```

Comment:
s/an doubling/a doubling/

Reply:
Good catch. I made a new PR for it since already merged it:

https://github.com/apache/couchdb-documentation/pull/166